sha1.js

```diff
@@ -8,29 +8,15 @@
  */
 
 var inherits = require('inherits')
-
 var Hash = require('./hash')
 
-var A = 0|0
-var B = 4|0
-var C = 8|0
-var D = 12|0
-var E = 16|0
-
-var W = new (typeof Int32Array === 'undefined' ? Array : Int32Array)(80)
-
-var POOL = []
+var W = new Array(80)
 
 function Sha1 () {
-  if(POOL.length)
-    return POOL.pop().init()
```

Comment:
This was removed from the other hash functions, if it is actually an improvement, maybe it should be in `Hash` instead.